packages/babel-plugin-transform-react-jsx/README.md

```diff
@@ -113,3 +114,9 @@ Note that the `@jsx React.DOM` pragma has been deprecated as of React v0.12
 `boolean`, defaults to `false`.
 
 When spreading props, use `Object.assign` directly instead of Babel's extend helper.
+
+### `throwIfNamespace`
+
+`boolean`, defaults to `true`.
+
+This option controls how to handle XML Namespaces. React's don't support it thus default behavior is to throw an error when XML namespace occurs, this behavior could be suppressed by setting it to false.
```

Comment:
Slight grammar nit:

`React's don't` -> `React's JSX doesn't`

Or maybe even:

> Toggles whether or not to throw an error if a XML namespaced tag name is used. For example:
> 
> ```jsx
> <f:image />
> ```
>
> Though the JSX spec allows this, it is _disabled by default_ since React's JSX does not currently  have support for it.

Reply:
@existentialism Thanks for the grammar 😥 . At the end, I've used your version. Providing an example is a good idea. 👍